rfcs/rfc-sec-20220316-1.md

```diff
@@ -0,0 +1,158 @@
+    * Asses threat
+        * Is it an actual vulnerability, is it a security feature etc?. Not everything reported as a vulnerability is a vulnerability. Generally, something is a vulnerability if it compromises data confidentiality, integrity, or availability. This may happen by enabling remote code execution, elevated permissions, or unintended access, but what separates a vulnerability from other unwanted behavior (a non-security bug) is a compromise in one or more of those areas.
+        * Identify initial impact level for issue.
+    * Discuss initial findings in private discord channel with SIR team.
+        * If an actual vulnerability, create a draft security advisory for issue, in the correct O3DE project/repository. See https://docs.github.com/en/code-security/security-advisories/creating-a-security-advisory for guidance.
+            * Update GHI with link to advisory link.
+            * Add SIR team members to draft advisory.
+            * Alert impacted SIGs via their chair(s) and the TSC. Add impacted chair(s) and TSC suggested contacts as as collaborators on security advisory: https://docs.github.com/en/github/managing-security-vulnerabilities/adding-a-collaborator-to-a-maintainer-security-advisory
+            * Provide a draft security response and request comments/changes from collaborators
+        * If not a vulnerability, thank reporter for their time and provide a reason why this is not a vulnerability. Categories should include “Working as intended”, “Bug” but not security related or a “Feature Request”. Let the reporter know reason for closure and direct them to security policy for future reports.
+            * Close issue in triage queue and let SIR team know issue is closed.
+    * Respond to issue reporter with information from triage and work with reporter to agree on embargo period/disclosure timeline. We should aim for disclosure with**90** days, even if no fix is available (For example https://vuls.cert.org/confluence/display/Wiki/Vulnerability+Disclosure+Policy). If there are conflicts in agreeing embargo period, please communicate with SIR team for next steps.
+
+##### Engage Resources to Mitigate Issue
+* Identify working group with impacted SIG(s) and SIR team. Group should contain at least one member of each impacted SIG,  the SIR team triage owner plus a SIG-release and TSC observer (if requested).
+        * Working group needs to setup reproduction of vulnerability.
+        * Working group needs to identify impact of vulnerability - wider impact, scope and severity
+        * Working group needs to attempt to provide mitigation in private fork: https://docs.github.com/en/articles/collaborating-in-a-temporary-private-fork-to-resolve-a-security-vulnerability to patch issue.
+ 
+##### Patch and Disclose Vulnerability 
```

Comment:
same [sub-bullet format](https://github.com/lmbr-pip/sig-security/blob/rfc_response/rfcs/rfc-sec-20220316-1.md#patch-and-disclose-vulnerability) comment as line 89.